src/main/java/org/jenkinsci/test/acceptance/docker/Docker.java

```diff
@@ -65,6 +66,26 @@ public boolean isAvailable() {
     }
 
     /**
+     * Checks if a given container is currently running.
+     *
+     * @param container Container id
+     * @return false if the container is not running, true otherwise
+     */
+    public boolean isContainerRunning(String container) throws IOException, InterruptedException {
+        ProcessBuilder pBuilder = cmd("ps").add("-q", "--filter", "\"id=" + container + '"').build();
+        Process psProcess = pBuilder.start();
+        psProcess.waitFor();
```

Comment:
So same concern as https://github.com/jenkinsci/acceptance-test-harness/pull/138#discussion_r68136355? Lemme know what the right way to do this is and I'll happily do so. =)